example/ck_tile/17_grouped_gemm/grouped_gemm.cpp

```diff
@@ -17,12 +17,16 @@
 #include "grouped_gemm.hpp"
 #include "utils.hpp"
 
+template <typename TType>
+constexpr std::size_t GetWorkspaceSize(const std::vector<TType>& gemm_descs)
```

Comment:
Why you have template here? This function should use this type: `grouped_gemm_kargs`. Moreover this won't be a `constexpr` function, as this information is run-time only. Additionally you need to have declaration of this function in `grouped_gemm.hpp` file.
```suggestion
std::size_t GetWorkspaceSize(const std::vector<grouped_gemm_kargs>& gemm_descs)
```